File: uri_online_judge/2289/2289.py

while True:

    st = input()

    X, Y = [int(x) for x in st.split(" ")]

    if X == 0 and Y == 0:
        break

    i = 0
    dist = 0

    while i < 64:
        if (X & (1 << i)) ^ (Y & (1 << i)):
            dist += 1
        i += 1

    print(dist)

File: uri_online_judge/2289/2289.cpp
#include <bits/stdc++.h>

using namespace std;

int main() 
{
    unsigned long long X, Y;

    while (scanf("%llu %llu", &X, &Y) && (X || Y)) {

        unsigned long long i = 0;
        int dist = 0;

        // cout << X << " " << Y;
        while (i < 64) {
            if ((X & (1ULL << i)) ^ (Y & (1ULL << i))) {
                dist++;
            }
            i++;
        }

        printf("%d\n", dist);
        // cout << dist << endl;
    }

    return 0;
}